src/main/java/jenkins/plugins/slack/StandardSlackService.java

```diff
@@ -88,6 +101,26 @@ public boolean publish(String message, String color) {
         return result;
     }
 
+    private String getTokenToUse() {
+        if (credentialsAuthTokenId != null && !credentialsAuthTokenId.isEmpty()) {
+            StringCredentials credentials = lookupCredentials(credentialsAuthTokenId);
+            if (credentials != null) {
+                logger.info("Using secret for Slack auth token.");
+                return credentials.getSecret().getPlainText();
+            }
+        }
+
+        logger.info("Using plain text for Slack auth token.");
```

Comment:
I see a lot of things in the screenshots.

If you want your log messages to illustrate the dichotomy of using either `authTokenCredentialId` or `authToken`, then that seems like a good path to me.